lint.sh

```diff
@@ -87,18 +88,23 @@ check_bash(){
   done < <(find ${fargs[@]} -iname \*.sh)
 }
 
-check_naming_standards() {
+# Check JJB for internal standards:
+#   * naming conventions
+#   * retention policy
```

Comment:
The above three lines look like they should be moved into the function. At the moment there are comments about the function either side the start of its definition.

Reply:
Done